src/battle/battle_4.c

```diff
@@ -117,6 +117,7 @@ extern u8 gEffectBank;
 extern u8 gAbsentBattlerFlags;
 extern u8 gMultiHitCounter;
 extern u16 gLastUsedMove[4];
+extern u16 gLastUsedMetronomeMove[4];
```

Comment:
Presumably there should be a non-externed definition somewhere? Or is that covered by the sys_ewram.txt?